pybleau/app/io/tests/test_plotting_roundtrip.py

```diff
@@ -90,14 +93,9 @@ def test_round_trip_df_plotter_cmap_scatter_type(self):
                               z_col_name="Col_4",
                               plot_config=config, plot_title="Plot 6")
 
-        analyzer = DataFrameAnalyzer(source_df=TEST_DF)
-
-        plot_manager = DataFramePlotManager(
-            contained_plots=[desc],
-            data_source=TEST_DF, source_analyzer=analyzer
-        )
-
-        self.assert_roundtrip_identical(plot_manager, ignore=self.ignore)
+        # Skipping the color_ attribute since it behaves strangely and is a
+        # proxy for the renderer color attribute anyway:
+        self.assert_df_plotter_roundtrip(desc, adtl_ignore={"color_"})
```

Comment:
`self.ignore` is already a list, so why is `assert_df_plotter_roundtrip` using anything but an additional list as its `ignore` argument?

Reply:
Sure. Fixed.